### DISCUSSION TITLE
Confused on a storage error

### DISCUSSION THREAD

**Discussion by KyleArthurs** [HomeSpan/HomeSpan#934] [Q&A]
i get an error in the code saying: 
/*
Sketch uses 1357737 bytes (103%) of program storage space. Maximum is 1310720 bytes.
Global variables use 55956 bytes (17%) of dynamic memory, leaving 271724 bytes for local variables. Maximum is 327680 bytes.
Sketch too big; see https://support.arduino.cc/hc/en-us/articles/360013825179 for tips on reducing it.
text section exceeds available space in board

Compilation error: text section exceeds available space in board
*/

but the error goes away when i delete a line that says 
/*
homeSpan.poll();
*/

can anyone help

**Comment by HomeSpan** [comment]
You need to select a larger partition scheme.  Please see the discussion at the top of the main HomeSpan readme page.

**Comment by KyleArthurs** [comment]
Thank you

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
